openassessment/xblock/static/js/src/oa_server.js

```diff
@@ -78,11 +77,9 @@ if (typeof OpenAssessment.Server == "undefined" || !OpenAssessment.Server) {
             element: The element to modify.
         **/
         renderLatex: function(element) {
-            $('.allow--latex', element).each(
-                function() {
-                    MathJax.Hub.Queue(['Typeset', MathJax.Hub, this]);
-                }
-            );
+            if(element.hasClass('allow--latex')) {
```

Comment:
Looking at the code I believe that the variable 'element' is not just a single element but an array of elements.
So instead of getting just first element by index ```element[0]``` you can write something like:
```javascript
element.filter(".allow--latex").each(function() {
    MathJax.Hub.Queue(['Typeset', MathJax.Hub, this]);
});
```